be/test/formats/orc/orc_lazy_load_test.cpp

```diff
@@ -0,0 +1,295 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2022-present, StarRocks Inc.
```

Comment:
You don't have to move orc lazy load test here. Stay it in apache-orc directory.